site/app/models/gradeable/Submitter.php

```diff
@@ -109,6 +109,14 @@ public function getRegistrationSection() {
         return $this->team_or_user->getRegistrationSection();
     }
 
+    /**
+     * Get the course_section_id for the submitter
+     * @return string
+     */
+    public function getCourseSectionId() {
```

Comment:
It's better to use native PHP type declarations when possible because PHP will enforce them in the development environment.

```suggestion
    /**
     * Get the course_section_id for the submitter
     */
    public function getCourseSectionId(): string {
```